sequelizeDatabase/sequelFunctions.js

```diff
@@ -264,54 +265,78 @@ insertFromCsvLine = (
     Comments,
     "\n"
   );
-  insertPerson(FundID, EmployeeName, LinkedInUrl, Comments)
```

Comment:
I expanded up while reviewing and can see now that this is part of the csv line insert code.